Description: Add documentation keys to systemd service files
Author: Felix Lechner <felix.lechner@lease-up.com>
Forwarded: no
Last-Update: 2017-12-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/svxlink/systemd/remotetrx.service.in
+++ b/src/svxlink/systemd/remotetrx.service.in
@@ -2,6 +2,7 @@
 
 [Unit]
 Description=SvxLink remote transceiver repeater control software
+Documentation=man:remotetrx(1)
 Requires=svxlink_gpio_setup.service
 After=network.target remote-fs.target syslog.target time.target
 After=svxlink_gpio_setup.service
--- a/src/svxlink/systemd/svxlink.service.in
+++ b/src/svxlink/systemd/svxlink.service.in
@@ -2,6 +2,7 @@
 
 [Unit]
 Description=SvxLink repeater control software
+Documentation=man:svxlink(1)
 Requires=svxlink_gpio_setup.service
 After=network.target remote-fs.target syslog.target time.target
 After=svxlink_gpio_setup.service
--- a/src/svxlink/systemd/svxlink_gpio_setup.service.in
+++ b/src/svxlink/systemd/svxlink_gpio_setup.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=SvxLink repeater control software GPIO setup
+Documentation=man:svxlink(1)
 
 [Service]
 Type=oneshot
